javascript/javascript1/week4/Highest-and-Lowest-Codewars-03.js

```diff
@@ -0,0 +1,9 @@
+function highAndLow(numbers) {
+  let numArray = numbers.split(" ");
```

Comment:
If you `console.log(numArray)` after this line, you will find that it's an array containing the numbers exactly as you would need to process with `Math.max` or `min`.

```
["8","3","-5","42","-1","0","0","-9","4","7","4","-4"]
```

In that sense, you don't need the next line with `map`
```
  let numList = numArray.map(Number);
```
And even you don't need to make a new variable, as you can just modify the received parameter
```suggestion
  numbers = numbers.split(" ");
```